include/deal.II/numerics/data_out_dof_data.templates.h

```diff
@@ -143,8 +156,19 @@ namespace internal
                 QIterated<dim>(QTrapezoid<1>(), n_subdivisions)));
             }
 
-          n_q_points = std::max(do_simplex ? quadrature1->size() : 0,
-                                do_hex ? quadrature2->size() : 1);
+          if (do_wedge)
+            {
+              Assert(1 <= n_subdivisions && n_subdivisions <= 2,
+                     ExcNotImplemented());
+              quadrature3.reset(new Quadrature<dim>(
```

Comment:
This is better done using
```
  quadrature3 = std::make_unique<Quadrature<dim>> (...)
```
Same above.

Separately, maybe name these variables `quadrature_hypercube`, `quadrature_simplex`, `quadrature_wedge`?